File: server/Models/Recipe.cs
namespace allspice_c.Models;

public class Recipe
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Title { get; set; }
    public string Category { get; set; }
    public string Instructions { get; set; }
    public string Img { get; set; }
    public string CreatorId { get; set; }

    // FIXME change string to Profile when the time comes
    public Profile Creator { get; set; }
}

File: client/src/models/Recipe.js


    export class Recipe{
        constructor(data){
            this.title = data.title
            this.category = data.category
            this.instructions = data.instructions
            this.id = data.id
            this.creator = data.creator
            this.img = data.img
            this.creatorId = data.creatorId
            this.createdAt = new Date(data.createdAt)
            this.updatedAt = new Date(data.updatedAt)
        }
    }